packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -715,13 +719,23 @@ export function TransferPanelMain({
                   forToken={nativeCurrency}
                   prefix={selectedToken ? '' : 'Balance: '}
                 />
-                {/* Only show ETH balance on parent chain */}
-                {isDepositMode && <ETHBalance balance={ethParentBalance} />}
+                {/* Only show ETH balance on L1 */}
```

Comment:
I think this should still say `on parent chain`?

Reply:
Updated in latest refactor